src/components/ProjectSearch/TextSearch.js

```diff
@@ -170,19 +194,13 @@ export default class TextSearch extends Component {
   }
 
   renderInput() {
-    const resultCount = this.getResultCount();
-    const summaryMsg = L10N.getFormatStr(
-      "sourceSearch.resultsSummary1",
-      resultCount
-    );
```

Comment:
could we keep summary msg here? it seems weird to store that?

Reply:
Something like this?
![screenshot_2](https://user-images.githubusercontent.com/23530054/32164652-6b381356-bd60-11e7-8ec8-bcef9504cfeb.png)

Reply:
i still don't see the need for storing `summaryMsg`, why can't we calculate each time we render?

Reply:
ah, I think because this would display the message before enter is pressed. Entering data into the field itself is not a sign of wanting to commit to a search and as such displaying the "0 Results message" does not seem to be the thing to do.

Reply:
hmm, can we have a check there to see that we are not currently searching?

Reply:
I think we could put a function that would construct the output based on the value of `this.updateCount`, similar to what SearchBar.js does. Something along the lines of:
![screenshot_3](https://user-images.githubusercontent.com/23530054/32197860-2b47c00a-bdc6-11e7-8b99-c7b9e883ef7d.png)